UPGRADING.md

```diff
@@ -18,6 +18,7 @@ This document will be updated to list important BC breaks and behavioral changes
  * removed all ``.class`` parameters, instead overwriting services via explicit Bundle configuration is preferred
  * renamed ``AbstractScalarParam::$array`` to ``AbstractScalarParam::$map``
  * added `ControllerTrait` for developers that prefer to use DI for their controllers instead of extending ``FOSRestController``
+ * when having an action called ``lockUserAction``, then it will have use the http method ``LOCK`` (RFC-2518) instead of ``PATCH``
```

Comment:
Might be an idea to think about this so that we could include it in 1.8. However, you will then still allow additional HTTP methods which one could consider a BC break too.